casper/src/test/scala/coop/rchain/casper/util/rholang/Interactive.scala

```diff
@@ -54,7 +50,7 @@ class Interactive private (runtime: Runtime[Task])(implicit scheduler: Scheduler
   def checkpointNames: List[String] = checkpoints.keys.toList
 
   def eval(code: String): Unit = {
-    TestUtil.eval(code, runtime, NormalizerEnv.Empty).runSyncUnsafe(Duration.Inf)
+    TestUtil.eval(code, runtime, Map.empty).runSyncUnsafe(Duration.Inf)
```

Comment:
Why do you prefer `Map.empty` over `NormalizerEnv.Empty`?

Reply:
I'm not sure if my reasoning holds water, please evaluate: Since `NormallizerEnv` is now type-safe - meaning it tracks the environment it contains, and, some pieces of code can't make use of it - I decided to split it. When you have to work with "just a map" you don't need to depend on `NormalizerEnv`. Only when you can take advantage of a precise `Env` type(like SD do), you should depend on `NornalizerEnv`. OTOH, it is possible to pass `NornalizerEnv.Empty` etc - but then you just type more (type var declaration, additional conversion to `.toEnv`) for no gain

Reply:
I guess that makes sense. Thank you for the explanation.